src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -89,7 +90,7 @@ protected function findTemplate($template, $throw = true)
         }
 
         if (false === $file || null === $file) {
-            throw new \Twig_Error_Loader(sprintf('Unable to find template "%s".', $logicalName), -1, null, $previous);
+            throw new \Twig_Error_Loader($errorMessage, -1, null, $previous);
```

Comment:
Not sure I understand. Instead of storing `$errorMessage = $e->getMessage();`, why not store the exception itself and throw it here?